src/components/Auth.tsx

```diff
@@ -0,0 +1,116 @@
+import { Button, Group, Modal, Space, Stack, Text, TextInput, List, ThemeIcon } from '@mantine/core';
+import { useContext, useEffect, useState } from 'react';
+import { showNotification } from '@mantine/notifications';
+import { Ad4minContext } from '../context/Ad4minContext';
+import { CircleCheck } from 'tabler-icons-react';
+
+type Props = {
+  info: string,
+  handleAuth: (auth: string) => void,
+}
+
+interface Capability {
+  with: Resource,
+  can: string[],
+}
+
+interface Resource {
+  domain: string,
+  pointers: string[],
+}
+
+const Auth = (props: Props) => {
+  const {
+    state: {
+      client
+    }
+  } = useContext(Ad4minContext);
+
+  const [opened, setOpened] = useState(false);
+
+  const authInfo = JSON.parse(props.info).auth;
+
+  useEffect(() => {
+    setOpened(true);
```

Comment:
Could set this to true by default and not have it changed when the component mounts.